BaseFilter/sections/specific.txt

```diff
@@ -346,6 +346,9 @@ $xmlhttprequest,third-party,denyallow=rsc.cdn77.org|linkvertise.com|fastly.net|s
 !
 ! NOTE: Specific rules
 !
+flightconnections.com##.free .display-box-2:not(#style_important)
+flightconnections.com###flc:not(#style_important)
+flightconnections.com##div:has(> .route-display-box-container)
```

Comment:
Maybe it could be more specific?
For example:
```adblock
flightconnections.com##.right-column:has(> .route-display-box-container)
```

Reply:
Done